lib/ansible/playbook/conditional.py

```diff
@@ -120,6 +126,12 @@ def _check_conditional(self, conditional, templar, all_vars):
         if conditional in all_vars and re.match("^[_A-Za-z][_a-zA-Z0-9]*$", conditional):
             conditional = all_vars[conditional]
 
+        delimiters = re.search('({{.*}}|{%.*%})', conditional)
```

Comment:
dont hardcode as the delimiters can be changed, the jinja2 environment object has the 'runtime' ones:

block_start_string, block_end_string, variable_start_string, variable_end_string, comment_start_string, comment_end_string